aslam_cv_frames/include/aslam/frames/visual-frame.h

```diff
@@ -101,7 +102,14 @@ class VisualFrame  {
 
   /// Get the number of keypoints measurements stored in this frame.
   inline size_t getNumKeypointMeasurements() const {
-    return static_cast<size_t>(getKeypointMeasurements().cols());
+    size_t num_measurements = 0;
+    if (hasKeypointMeasurements()) {
```

Comment:
can we just get rid of the if statements and add `0` in cases where it does not apply

Reply:
I don't think you can. The channels are stored inside a map so if a channel is not initialized then it will actually not exist and instead truing to call `getKeypointMeasurements` will result in a CHECK failure.

Reply:
I was afraid that it would do this.